src/compas/datastructures/tree/tree.py

```diff
@@ -379,11 +377,9 @@ def traverse(self, strategy="depthfirst", order="preorder"):
         ----------
         strategy : {"depthfirst", "breadthfirst"}, optional
             The traversal strategy.
-            Default is ``"depthfirst"``.
 
```

Comment:
there can't be a blank line in parameter lists

Reply:
oops.... fixed